src/rdf4cpp/Timezone.hpp

```diff
@@ -130,23 +132,144 @@ struct Timezone {
 
 using OptionalTimezone = std::optional<Timezone>;
 
-using TimePoint = std::chrono::time_point<std::chrono::local_t, std::chrono::milliseconds>;
+template<typename I = int64_t>
```

Comment:
Since we are only using `int64_t` it would make sense to remove the template

Reply:
test cases use `int`. also keeping the template makes it easier to adjust in the future.

Reply:
Since we will not be using `int` in the real world I would suggest to change the tests to `uint64_t`. Regarding the template: I don't think we'll need to change it in the future